### PR TITLE
chore: support specifying postgres SSL CA certs and client certs/keys

### PR DESCRIPTION
**Pull request by lucasmarshall**
## Test Plan

Tested against a GCP Cloud SQL postgres server configured to require SSL, and also tested with it configured to require client certs.

### CODE REVIEW

File: apps/mgmt-ui/src/components/connectors/destination/PostgresDestinationDetailsPanel.tsx [hunk 1]
```diff
@@ -253,15 +269,100 @@ export default function PostgresDestinationDetailsPanel({ isLoading }: PostgresD
...
             }}
             value={sslMode ?? 'disable'}
-            options={[{ value: 'disable' }, { value: 'allow' }, { value: 'prefer' }, { value: 'require' }]}
+            options={[
+              { value: 'disable' },
+              { value: 'no-verify' },
+              { value: 'prefer' },
+              { value: 'require' },
+              { value: 'verify-ca' },
+              { value: 'verify-full' },
```
Comment (tomkit) [inline apps/mgmt-ui/src/components/connectors/destination/PostgresDestinationDetailsPanel.tsx:283]: have we tested all of these combinations? as well as on AWS RDS?

Comment (lucasmarshall) [reply apps/mgmt-ui/src/components/connectors/destination/PostgresDestinationDetailsPanel.tsx:283]: Yes, I tested them all. I have not tested RDS, do we have a customer that is needing this there?

Comment (lucasmarshall) [reply apps/mgmt-ui/src/components/connectors/destination/PostgresDestinationDetailsPanel.tsx:283]: Tested CA certs on RDS and it works. RDS doesn't support client certs.

**Review by tomkit — APPROVED** [review]
Looks like some build issues

**Pull request merged**